py/torch_tensorrt/dynamo/fx_ts_compat/lower.py

```diff
@@ -70,7 +71,9 @@ def compile(
         timing_cache_prefix: Timing cache file name for timing cache used by fx2trt.
         save_timing_cache: Update timing cache with current timing cache data if set to True.
         cuda_graph_batch_size: Cuda graph batch size, default to be -1.
-        use_experimental_fx_rt: Uses the next generation TRTModule which supports both Python and TorchScript based execution (including in C++).
+        use_python_runtime: Whether to strictly use Python runtime or C++ runtime. To auto-select a runtime
+            based on C++ dependency presence (preferentially choosing C++ runtime if available), leave the
+            argument as None
```

Comment:
Added a new compilation argument called `use_python_runtime` to replace `use_experimental_rt` and `use_experimental_fx_rt`. This new compilation argument has three potential settings/implications:

- `True`: Strictly use Python runtime
- `False`: Strictly use C++ runtime
- `None` [**DEFAULT**]: Automatically detect presence of C++ dependency and select C++ runtime if available, else default to Python runtime